packages/devtools_app/lib/html_main.dart

```diff
@@ -65,11 +65,12 @@ void main() {
         return;
       }
 
-      // Show the Q3 DevTools survey. Stop showing the survey after
-      // November 14, 2019 (4 weeks after the survey start date
-      // October 17, 2019).
-      final surveyThresholdDate = DateTime(2019, 11, 14);
-      if (DateTime.now().isBefore(surveyThresholdDate)) {
+      // Show the Q1 2020 DevTools survey if the current date falls in range.
+      final surveyStartDate = DateTime(2020, 3, 11);
+      final surveyEndDate = DateTime(2020, 4, 10);
+      final now = DateTime.now();
+      if (now.isAfter(surveyStartDate) && now.isBefore(surveyEndDate)) {
+        await ga.setActiveSurvey('Q1-2020');
```

Comment:
setActiveSurvey can return false if any error creating .devtiols first time, persisting the properties or filesystem problem.  Nothing specific just success or failure.

Reply:
added a check for this. thanks